### PR TITLE
docs: avoid code in heading

### PR DESCRIPTION
**Pull request by snitin315**
Avoid `code(``)` in heading as it has different font family and also for consistency throughout the docs

### CODE REVIEW

**Comment by chenxsan** [comment]
There're many others, can you replace them all in this pull request? It won't be a problem to downstream repositories as far as I can see.

**Comment by snitin315** [comment]
I left `performance.mdx` as it will be updated in https://github.com/webpack/webpack.js.org/pull/5295

**Comment by snitin315** [comment]
I don't know why CI is failing 😞

**Comment by chenxsan** [comment]
[The error](https://github.com/webpack/webpack.js.org/pull/5297/checks?check_run_id=3331787470#step:5:615) isn't helpful. Maybe you can push just one commit every time to isolate the change causing the problem.

**Comment by snitin315** [comment]
looking into it 👍🏻

**Comment by snitin315** [comment]
Ok, I think it's somehow related to `mdx`, without code, in `test.{option}` `{option}` is being considered as a variable I guess. I will revert such headings for now.

**Comment by snitin315** [comment]
Now something is wrong with the last commit https://github.com/webpack/webpack.js.org/pull/5297/commits/ffb32e94b65ad6fc2c46caa7990fc120ef066ff4, the error isn't helpful as well.

File: src/content/api/compilation-hooks.mdx [hunk 1]
```diff
@@ -47,293 +47,293 @@ compilation.hooks.buildModule.tap(
...
+### afterOptimizeTree
 
 `SyncHook`
 
 Called after the dependency tree optimization has completed with success.
 
 - Callback Parameters: `chunks` `modules`
 
-### `optimizeChunkModules`
+### optimizeChunkModule
```
Comment (snitin315) [inline src/content/api/compilation-hooks.mdx:172]:
```suggestion
### optimizeChunkModules
```

File: src/content/api/compilation-hooks.mdx [hunk 2]
```diff
@@ -47,293 +47,293 @@ compilation.hooks.buildModule.tap(
...
+### optimizeChunks
 
 `SyncBailHook`
 
 Called at the beginning of the chunk optimization phase. A plugin can tap into this hook to perform optimizations on chunks.
 
 - Callback Parameters: `chunks`
 
-### `afterOptimizeChunks`
+## afterOptimizeChunks
```
[the remark this answers is not in the document]
Comment (snitin315) [reply src/content/api/compilation-hooks.mdx]: Thanks, CI is green now.

**Review by chenxsan — APPROVED** [review]
Looks good. Thanks for the hard work!

**Pull request merged**